.babelrc

```diff
@@ -1,11 +1,9 @@
 {
   "presets": [
-    ["es2015", { "modules" : false }],
-    "react",
-    "stage-1"
-  ],
-  "plugins": [
-    "transform-runtime"
+    "transform-runtime",
+    "es2015",
+    "stage-0",
```

Comment:
Be careful with using stage-0, the syntax/proposal/... can completely change: https://tc39.github.io/process-document/

I'm not saying you shouldn't use it, just know that it can have an impact later